packages/components/src/components/Input/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import { VisuallyHidden } from 'reakit/VisuallyHidden';
+import { uniqueId } from 'lodash-es';
+
+const placeholderStyles = {
+  color: 'input.placeholderForeground',
+  fontSize: 13,
```

Comment:
```suggestion
  fontSize: 3,
```

Reply:
The token is the array index

OHHHHHHHHHHHHH

Reply:
I should probably right the keys down 😅